apps/charterafrica/src/components/Footer/Footer.js

```diff
@@ -17,7 +21,17 @@ const Footer = React.forwardRef(function Footer(props, ref) {
     logo,
     projectDescription,
     siteDescription,
+    subscription,
   } = props;
+  const projectDesc = serialize(projectDescription.children);
+  const siteDesc = serialize(siteDescription.children);
+
+  const [toEmbed, setToEmbed] = useState("");
+
+  useEffect(() => {
+    setToEmbed(subscriptionEmbedCode);
+  }, []);
```

Comment:
Any reason this needs `useEffect`?

Reply:
Yes,

This is a workaround for hydration mismatch error that comes with embeding html.

This fix has been documented here https://nextjs.org/docs/messages/react-hydration-error